lego_ee_tf: "/transforms/lego_ee.tf"
lego_world_tf: "/transforms/lego_world.tf"
data_dir: "/data/"
block_type: "4x1"
x_range:
  - -0.005
  - 0.005
y_range: 
  - -0.005
  - 0.005
theta_range: 
  - -10.0
  - 10.0
approach_height: 0.03
move_down_velocity_scaling: 0.01
lift_height: 0.01
pull_up_velocity_scaling: 0.001
place_rotation: 25
pick_rotation: 25
fts_detector:
  topic_name: '/fts'
  force_threshold: 0.5
lego_detector:
  topic_name: '/side_camera/color/image_cropped'
  top_bbox:
    - 900
    - 300
    - 1100
    - 450
  bot_bbox:
    - 900
    - 450
    - 1100
    - 575
  detection_threshold: 0.8
rosbag_data_recorder:
  topics:
    - /side_camera/color/image_cropped
    - /camera/color/image_raw
    - /namespace/joint_states
    - /fts
    - /audio
    - /audio_info
    - /namespace/terminator/skill_termination_signal
    - /outcome/lego_detector
    - /outcome/fts_detector
